src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/languageConstructions/UnnecessaryElseInspector.java

```diff
@@ -0,0 +1,95 @@
+package com.kalessil.phpStorm.phpInspectionsEA.inspectors.languageConstructions;
+
+import com.intellij.codeInspection.LocalQuickFix;
+import com.intellij.codeInspection.ProblemHighlightType;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiElementVisitor;
+import com.jetbrains.php.lang.lexer.PhpTokenTypes;
+import com.jetbrains.php.lang.psi.elements.*;
+import com.kalessil.phpStorm.phpInspectionsEA.fixers.UnnecessaryElseFixer;
+import com.kalessil.phpStorm.phpInspectionsEA.openApi.BasePhpElementVisitor;
+import com.kalessil.phpStorm.phpInspectionsEA.openApi.BasePhpInspection;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * (c) Funivan <alotofall@gmail.com>
+ */
+public class UnnecessaryElseInspector extends BasePhpInspection {
+    private static final String message = "Keyword else can be safely removed.";
+
+    @NotNull
+    public String getShortName() {
+        return "UnnecessaryElseInspection";
+    }
+
+
+    @Override
+    @NotNull
+    public PsiElementVisitor buildVisitor(@NotNull final ProblemsHolder holder, boolean isOnTheFly) {
+        return new BasePhpElementVisitor() {
+            @Override
+            public void visitPhpElse(Else elseStatement) {
+                super.visitPhpElse(elseStatement);
+                visitElseKeywords(elseStatement);
+            }
+
+            @Override
+            public void visitPhpElseIf(ElseIf elseIfStatement) {
+                super.visitPhpElseIf(elseIfStatement);
+                visitElseKeywords(elseIfStatement);
+            }
+
+            private void visitElseKeywords(Statement elseStatement) {
```

Comment:
I still have method `hasBraces` to check if we try to handle code without braces.  Still do not know how to check this with `ExpressionSemanticUtil.getGroupStatement`